Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md

```diff
@@ -60,35 +60,41 @@ To verify your policy was assigned correctly:
 
-1. Set up SSL certificate and upload to the cloud service
+2. Set up SSL certificate and upload to the cloud service
     1. Create a wildcard certificate for your service. This certificate should not be a self-signed certificate. For instance, if your bot is hosted at `bot.contoso.com`, create the certificate for `*.contoso.com`.
     2. Upload the certificate to the cloud service.
     3. Copy the thumbprint for later.
 
-1. Set up cloud service configuration
+3. Set up cloud service configuration
     1. Open powershell, go to the folder that contains file `configure_cloud.ps1`. The file is in the `Samples` directory.
+
     2. Run the powershell script with parameters:
-        * `> .\configure_cloud.ps1 -p .\BetaSamples\LocalMediaSamples\PolicyRecordingBot\ -dns {your DNS name} -cn {your CN name, should be the same as your DNS name} -thumb {your certificate thumbprint} -bid {your bot name} -aid {your bot app id} -as {your bot secret}`, for example `.\configure_cloud.ps1 -p .\BetaSamples\LocalMediaSamples\PolicyRecordingBot\ -dns bot.contoso.com -cn bot.contoso.com -thumb ABC0000000000000000000000000000000000CBA -bid bot -aid 3853f935-2c6f-43d7-859d-6e8f83b519ae -as 123456!@#$%^`
+        `> .\configure_cloud.ps1 -p {path to project} -dns {your DNS name} -cn {your CN name, should be the same as your DNS name} -thumb {your certificate thumbprint} -bid {your bot name} -aid {your bot app id} -as {your bot secret}`
+        
+        For example:
+        
+         `.\configure_cloud.ps1 -p .\V1.0Samples\LocalMediaSamples\PolicyRecordingBot\ -dns bot.contoso.com -cn bot.contoso.com -thumb ABC0000000000000000000000000000000000CBA -bid bot -aid 3853f935-2c6f-43d7-859d-6e8f83b519ae -as 123456!@#$%^`
 
-1. Publish the bot from VS:
+4. Publish the bot from VS:
     1. Right click PolicyRecordingBot, then click `Publish...`. Publish it to the cloud service you created earlier.
 
 ### Test
 
 1. Set up the test meeting and test clients:
    1. Sign in to Teams client with a non-recorded test tenant user.
-   1. Use another Teams client to sign in with the recorded user. (You could use an private browser window at https://teams.microsoft.com)
+   2. Use another Teams client to sign in with the recorded user. (You could use an private browser window at https://teams.microsoft.com). Use Teams client if you couldn't see the call notification by using Teams browser for some reason.
 
-1. Place a call from the Teams client with the non-recorded user to the recorded user.
+2. Place a call from the Teams client with the non-recorded user to the recorded user.
 
-1. Your bot should now receive an incoming call, and join the call (See next step for retrieving logs). Use the recorded user's Teams client to accept the call.
+3. Your bot should now receive an incoming call, and join the call (See next step for retrieving logs). Use the recorded user's Teams client to accept the call. A banner should show up once the call get established.
```

Comment:
Rephrase this as

Your bot should receive the incoming call. Use the recorded users' Teams client to accept the call. Once the call is established, you'll see a banner indicating that the recording has started. See the next step to learn how you can retrieve the call log.

Reply:
Change it to 
```
Your recording bot should receive the incoming call and join the call immediately. Use the recorded users' Teams client to accept the call. Once the P2P call is established, you'll see a banner indicating that the recording has started. See the next step to learn how you can retrieve the call log.
```